drivers/ethernet/Kconfig

```diff
@@ -49,6 +49,12 @@ config ETH_QEMU_EXTRA_ARGS
 	  spaces between arguments. Example: "mac=02:03:04:f0:0d:01" or
 	  "mac=02:03:04:f0:0d:01,downscript=no"
 
+config ETH_QBV
+	bool "QBV support"
+	depends on PTP_CLOCK
+	help
+	  Enable QBV support.
```

Comment:
More information about what Qbv is is needed here.
Why it depends on PTP?

As the Qbv is a generic Ethernet feature, I feel it should probably be introduced in `subsys/net/l2/ethernet/Kconfig` instead of the driver config.
Please also call it Qbv in the text, for Kconfig option using QBV is ok.